frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -327,7 +319,19 @@ const DayLabel = styled.p<DayLabelType>`
 const Star = styled.img`
   height: 51px;
```

Comment:
so sorry for being nit picky.. The star sizing at like 900px screen size goes to the edge of the container. Maybe instead of px use vw/vh ?

<img width="390" alt="Screen Shot 2021-03-21 at 10 37 53 PM" src="https://user-images.githubusercontent.com/17476012/111933543-14957a00-8a96-11eb-9a26-67887220a532.png">

Reply:
This has been updated!